### PR TITLE
Cl refactor m1

### PR DESCRIPTION
**Pull request by clangenb**

### CODE REVIEW

**Comment by clangenb** [comment]
successfully run substraTEE demo M2 after refactoring

File: src/examples/extrinsic/crypto.rs [hunk 1]
```diff
@@ -0,0 +1,113 @@
+use bip39::{Language, Mnemonic, MnemonicType};
```
Comment (electronix) [inline src/examples/extrinsic/crypto.rs]: Copyright header fehlt

File: src/json_rpc/client.rs [hunk 2]
```diff
@@ -0,0 +1,101 @@
+use ws::{CloseCode, Handler, Handshake, Message, Result, Sender};
```
Comment (electronix) [inline src/json_rpc/client.rs]: Copyright header missing

File: src/examples/extrinsic/crypto.rs (hunk 1 above)
Comment (clangenb) [reply src/examples/extrinsic/crypto.rs]: done in other branch

**Pull request merged**